### DISCUSSION TITLE
There should be an option to control what shows up.

### DISCUSSION THREAD

**Discussion by basilaiman** [community/community#32873] [Other Feature Feedback, Questions, & Ideas]
Let me start by saying that the feed is useful for discovering new repo and projects. 

But there should be more control over that.  

There should be control over who can see my latest activity. I need to control what to share.

It's not comfortable that random people I don't know see what repositories I just starred.

**Comment by HossamMuhammedOmar** [comment]
**Yes I agree with you**

**Comment by ju5t** [comment]
It should be the other way round too. For some reason I'm always seeing random peoples actions, even when I remove some of the relevant filter options they keep showing up. It's weird and I don't want it.

**Comment by lxvx** [comment]
Agreed- it would make github a way better place
